### DISCUSSION TITLE
67_maxar_open_data

### DISCUSSION THREAD

**Discussion by emilianosantin** [opengeos/leafmap#930] [Q&A] [answered]
Hi 

How do we get the URL for other disasters for instance Helene or Milton?

Make the mosaic json file available on the web, then you can add the mosaic to the map:
```
m = leafmap.Map()
url = "https://geospatial.glitch.me/1050050044DE7E00.json" #this url 
m.add_stac_layer(url, name="Mosaic")
m.add_gdf(gdf, layer_name="Footprints")
m
```

And also how do we obtain to add the footprints to the map:

```
m = leafmap.Map(center=[36.844461, 37.386475], zoom=8)
# m.add_gdf(gdf, layer_name="Footprints")
url = "https://cdn.glitch.global/cc5b7737-d8d0-4b07-bf2f-867b9009e986/maxar_footprints.geojson?v=1676583955698" # this url
m.add_geojson(url, layer_name="Footprints")
m
```

Thanks for your support

**Comment by giswqs** [comment, marked as answer]
See 

- https://github.com/opengeos/maxar-open-data/tree/master/datasets/HurricaneHelene-Oct24
- https://github.com/opengeos/maxar-open-data/tree/master/datasets/HurricaneMilton-Oct24
